packages/teams-js/src/public/media.ts

```diff
@@ -174,7 +198,7 @@ export namespace media {
    * Media object returned by the select Media API
    */
   export class Media extends File {
-    constructor(that: Media = null) {
```

Comment:
> = null [](http://example.com/codeflow?start=27&length=7)

`null` was not a legal value for this. Since the value is only used immediately below (line 203) and already handles the `undefined` case the same as `null,` rather than using a default value of `null`, which would happen if `that` wasn't passed, it now instead will set it to `undefined` if `that` wasn't passed, which is handled just the same as before.